### PR TITLE
Add inline documentation

### PR DESCRIPTION
**Pull request by klaaspieter**

### CODE REVIEW

**Comment by klaaspieter** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;8v6bImg4&#x2F;95-inline-api-documentation"> Inline API documentation</a>

File: lib/courier/client.rb [hunk 1]
```diff
@@ -2,15 +2,56 @@
...
+    # Send a notification to all devices registered with a channel
+    #
+    # @param [String] channel
+    #        name of the channel. Names are automatically parameterized
+    #        using
+    #        [ActiveSupport::Inflector](http://apidock.com/rails/ActiveSupport/Inflector).
+    #
+    # @param {String: String, Hash, Array} payload
+    #        The
+    #        [APNs](https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/TheNotificationPayload.html#//apple_ref/doc/uid/TP40008194-CH107-SW1) payload to broadcast. See [lowdown](http://www.rubydoc.info/gems/lowdown/Lowdown%2FNotification%3Aformatted_payload) documentation for more information on how the payload is processed.
```
Comment (calleerlandsson) [inline lib/courier/client.rb]: If it is possible to break this line, that'd be great.

File: lib/courier/client.rb [hunk 2]
```diff
@@ -2,15 +2,57 @@
 require "active_support/inflector"
 
 module Courier
+
+  # Client object manages authenticating and communicating with the Courier API.
   class Client
+
+    # The Courier API base URL.
+    # @!visibility private
     DEFAULT_BASE_URL = "https://courier.thoughtbot.com".freeze
```
Comment (calleerlandsson) [inline lib/courier/client.rb]: Should this constant be made private with `private_constant`?

**Comment by calleerlandsson** [comment]
I haven't written a ton of yard docs in the past but I think this looks good.

**Pull request merged**